shared/cfg/base.cfg

```diff
@@ -620,6 +624,22 @@ migrate_proto = "tcp"
 # Remember to open ports on both source and destination
 migrate_port = 49152
 
+##### migration_template use only parameters
+
+# Whether to migrate vm back to src host, default to yes 
+# migrate_vm_back = "yes"
+
+# virsh migrate options, default to "--live --p2p"
+# virsh_migrate_options = '--live --p2p'
+
+# Protocol used in migrate desturi, default to tcp
+# remote_protocol = 'tcp'
```

Comment:
rename remote_protocol to migrate_remote_protocol ?

Reply:
Actually, this remote_protocol is libvirtd remote access protocol.
I don't want to mix it with "migration protocol" which is used for migration data transfer.

Reply:
How about libvirtd_remote_access_protocol?

Reply:
Changed to migrate_desturi_proto to make it more strait